File: Service/Implement/DanhMucCompanyTinhTrangService.cs
namespace Service.Implement
{
    public class DanhMucCompanyTinhTrangService : BaseService<DanhMucCompanyTinhTrang, IDanhMucCompanyTinhTrangRepository>
    , IDanhMucCompanyTinhTrangService
    {
    private readonly IDanhMucCompanyTinhTrangRepository _DanhMucCompanyTinhTrangRepository;
    public DanhMucCompanyTinhTrangService(IDanhMucCompanyTinhTrangRepository DanhMucCompanyTinhTrangRepository) : base(DanhMucCompanyTinhTrangRepository)
    {
    _DanhMucCompanyTinhTrangRepository = DanhMucCompanyTinhTrangRepository;
    }
    }
    }

File: CMS/src/app/shared/DanhMucCompanyTinhTrang.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucCompanyTinhTrang } from 'src/app/shared/DanhMucCompanyTinhTrang.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucCompanyTinhTrangService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucCompanyTinhTrang";
    }
}

